### PR TITLE
(Draft) - Initial review

### PR DESCRIPTION
**Pull request by josevarela**
## Purpose
Draft PR to review pre-populated repo content

* do not merge *
* ...

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X] Other... Please describe:
    code review
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# App Services sample workload
+
+It is a simple web application which is able to connect a SQL Dadatabase.
```
Comment (josevarela) [inline README.md]:
```suggestion
This GitHub repository contains a simple web application sample that connects to an Azure SQL Database. The sample application supports reference implementations of articles published in the Azure Architecture Center (AAC).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+It supports App Service references implementations.
```
Comment (josevarela) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- App Service Simple Web App (coming soon)
+- [App Service Baseline implementation](https://github.com/Azure-Samples/app-service-baseline-implementation)
```
Comment (josevarela) [inline README.md]:
```suggestion
- [Baseline highly available zone-redundant web application](https://github.com/Azure-Samples/app-service-baseline-implementation)
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## Features
+
+This project framework provides the following features:
```
Comment (josevarela) [inline README.md]:
```suggestion
The sample project provides the following features:
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Home Page
```
Comment (josevarela) [inline README.md]:
```suggestion
- A simple Welcome page
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  ![Home Page](homePage.png)
+- A page which show database Product Categories to show connectivity
```
Comment (josevarela) [inline README.md]:
```suggestion
- A product categories page to illustrate database connectivity
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+## Getting Started
+
+### Prerequisites
+
+- [.Net 6.0](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [Visual Studio Community 2022](https://visualstudio.microsoft.com/vs/community/)
+- [Adventureworks database example](https://learn.microsoft.com/sql/samples/adventureworks-install-configure)
+
+### Installation
```
Comment (josevarela) [inline README.md]:
```suggestion
### To edit in Visual Studio
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+1. Install [Adventureworks database example](https://learn.microsoft.com/sql/samples/adventureworks-install-configure)
+1. Get connnetion string and add it on appsettings.json
```
Comment (josevarela) [inline README.md]:
```suggestion
1. Get connnetion string and add it to appsettings.json
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+1. Open SimpleWebApp.sln with Visual Studio
+
+> :warning: there is a already compiled version (SimpleWebApp.zip) which is going to be used in the reference implementations.
```
Comment (josevarela) [inline README.md]:
```suggestion
> :warning: the repo contains an already compiled version (SimpleWebApp.zip) of the solution which is going to be used in the reference implementations.
```

Comment (josevarela) [inline README.md]: Note for discussion:

I'd like to get rid of the zip file from this repo if possible. Let's discuss how to best handle the scenario

**Review by josevarela — COMMENTED** [review]
Some suggestions in-line

**Review by v-fearam — APPROVED** [review]

**Pull request closed without merging**